lib/folio_sync/archives_space/marc_exporter.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module FolioSync
+  module ArchivesSpace
+    class MarcExporter
+      ONE_DAY_IN_SECONDS = 24 * 60 * 60
```

Comment:
I think this can be removed since it's no longer used in this file.

Reply:
Updated!